spec/std/json/serializable_spec.cr

```diff
@@ -35,6 +35,16 @@ class JSONAttrPerson
   end
 end
 
+struct JSONAttrPersonWithTwoFieldInInitialize
```

Comment:
Add a test for single field too.

Reply:
there is many classes with 1 arg already